transformers4rec/torch/trainer.py

```diff
@@ -106,7 +106,6 @@ def __init__(
     ):
 
         mock_dataset = DatasetMock()
-        hf_model = HFWrapper(model)
```

Comment:
The wrapper is not needed as we are overriding all the HF Trainer methods that call the model's forward method: `prediction_step` and `compute_loss`